rest/src/helpers/or_reject.rs

```diff
@@ -31,3 +31,10 @@ impl<T> OrReject<T> for anyhow::Result<T> {
         self.map_err(|e| reject::custom(RestError::Request(e.to_string())))
     }
 }
+
+impl<T> OrReject<T> for Option<T> {
+    /// Returns the result if it is successful, otherwise returns a rejection.
+    fn or_reject(self) -> Result<T, Rejection> {
```

Comment:
maybe there could be a `&str` param here indicating the reason behind the rejection?

Reply:
This suggestion deserves a separate PR because it implies updating the implementation of `OrReject` (and all the places it's called).